Core/include/Acts/Seeding/SeedFinderUtils.ipp

```diff
@@ -147,6 +147,30 @@ inline void transformCoordinates(Acts::SpacePointData& spacePointData,
   }
 }
 
+inline std::array<float, 6> transformCoordinates(
+    const std::array<float, 4>& lineCircleVariables) {
+  auto& [deltaX, deltaY, cosPhiM, sinPhiM] = lineCircleVariables;
```

Comment:
why passing an array if then you immediately take the difference components? Why not requiring 4 floats as inputs to the function?